_examples/json/main.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/alecthomas/participle"
+)
+
+type pathExpr struct {
+	Parts []part `@@ { "." @@ }`
+}
+
+type part struct {
+	Obj string `@Ident`
+	Acc []acc  `("[" @@ "]")*`
+}
+
+type acc struct {
+	Name  *string `@(String|Char|RawString)`
```

Comment:
Is this correct? Would `@String` be sufficient?

Reply:
Yep `@String` should be sufficient.